package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "aind-metadata-entry",
-  "version": "0.1.6",
+  "version": "0.2.0",
```

Comment:
The version bump github action bumped the version but did not update the package-lock.json. I'll create a separate ticket for this